app/vmalert/rule/alerting.go

```diff
@@ -733,3 +637,33 @@ func (ar *AlertingRule) alertsToSend(ts time.Time, resolveDuration, resendDelay
 	}
 	return alerts
 }
+
+// GetAlerts returns active alerts of rule
+func (ar *AlertingRule) GetAlerts() []*notifier.Alert {
```

Comment:
nit: would be nice to list public methods in the beginning of the file, so it would be clear for a reader what this package is about

Reply:
Moved)